db/migrate/20241007210254_add_medical_expenses_to_nj_intake.rb

```diff
@@ -0,0 +1,5 @@
+class AddMedicalExpensesToNjIntake < ActiveRecord::Migration[7.1]
+  def change
+    add_column :state_file_nj_intakes, :medical_expenses, :integer, default: 0, null: false
```

Comment:
[dust] I see we're doing integer here for this money field, which is fine given our discussion yesterday but will be another field to track for updating if we switch to accepting non-rounded numbers?

Reply:
@aloverso @mmazanec22 do we have a ticket/notepad tracking where we're using non-rounded numbers somewhere?

Reply:
Made a ticket https://github.com/newjersey/affordability-pm/issues/98